packages/mcfunction/src/parser/entry.ts

```diff
@@ -52,3 +52,12 @@ export function entry(
 const comment = core.comment({
 	singleLinePrefixes: new Set(['#']),
 })
+
+/**
+ * Concatenates lines together on trailing backslashes before running the main
+ * {@link mcfunction} parser
+ */
+export const entry = (
+	commandTreeName: string,
+	argument: ArgumentParserGetter,
+) => core.concatOnTrailingBackslash(mcfunction(commandTreeName, argument))
```

Comment:
This should only be enabled for a certain version. I think we could either add a new parameter to the `entry` function `supportsBackslashContinuation` to toggles the `core.concatOnTrailingBackslash()` wrap, or move the wrap to `java-edition/mcfunction`.

Reply:
done in [`c16a08a` (#1180)](https://github.com/SpyglassMC/Spyglass/pull/1180/commits/c16a08a191a2d157d981528a4ab7d1efd782ec4a)

Reply:
the flag param is there now, but i think i'd prefer to do the actual version checking -> gating to after `1.20.2` in a separate PR since this one's already pretty big at this point